__test__/app.test.js

```diff
@@ -9,18 +9,46 @@ const app = require('../app');
 describe('/api', () => {
   afterAll(() => connection.destroy());
   beforeEach(() => connection.seed.run());
-
-  // Ayako BRANCH OUT FOR EACH REQUEST!! DON'T WORK ON MASTER
-  describe('/food-items', () => {
-    it('GET 200 - responds with an array of all food_items', () => {
+// Ayako BRANCH OUT FOR EACH REQUEST!! DON'T WORK ON MASTER 
+  describe('GET /api/food-items', () => {
+    it('SUCCESS status 200 - responds with an array of all food_items', () => {
       return request(app)
-        .get('/api/food-items')
-        .expect(200)
-        .then(({ body }) => {
-          expect(body.foodItems).toHaveLength(21);
-        });
-    });
-  });
+      .get('/api/food-items')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.foodItems).toHaveLength(21)
```

Comment:
Why is there a length?

Reply:
FoodItem objects are stored in an array and it will look like [{food1},{food2},...] and so on. Counting the length on the array will prove all the food item is passed or not with the GET method.